pytravharv/pytravharv.py

```diff
@@ -0,0 +1,131 @@
+import logging
+import logging.config
+import os
+from typing import List, Optional
+
+import yaml
+from pyrdfstore import create_rdf_store
+from rdflib import Graph
+
+from pytravharv.common import QUERY_BUILDER, insert_resource_into_graph
+from pytravharv.store import TargetStoreAccess as RDFStoreAccess
+from pytravharv.TravHarvConfigBuilder import (
+    TravHarvConfig,
+    TravHarvConfigBuilder,
+)
+from pytravharv.TravHarvExecuter import TravHarvExecutor
+
+# log = logging.getLogger("pyTravHarv")
+log = logging.getLogger(__name__)
+
+
+class TravHarv:
+    """Assert all paths for given subjects.
+    Given a configuration file, assert all paths for all subjects in the configuration file.
+    :param config_folder: str
+    :param name: str
+    :param output: str
+    :param context: list[str]
+    :param target_store_info: Optional[list[str]]
+    :param verbose: bool
+    """
+
+    def __init__(
+        self,
+        config_folder: str,
+        name: Optional[str] = None,
+        output: Optional[str] = None,
+        context: Optional[List[str]] = None,
+        target_store_info: Optional[List[str]] = None,
+        verbose: bool = True,
+    ):
+        """Assert all paths for given subjects.
+        Given a configuration file, assert all paths for all subjects in the configuration file.
+        :param config_folder: str
+        :param name: str
```

Comment:
name?

taskname? jobname? operational name?  relation to the config?

this is the thing that gets translated to the operational context that also results in the named_graph ?